ql/cashflows/fixedratecoupon.cpp

```diff
@@ -186,8 +186,11 @@ namespace QuantLib {
                                 start, end, start, end, exCouponDate));
             leg.push_back(temp);
         } else {
-            Date ref = end - schedule_.tenor();
-            ref = schCalendar.adjust(ref, schedule_.businessDayConvention());
+            BusinessDayConvention bdc = schedule_.businessDayConvention();
+            Date ref = schedule_.calendar().advance(end,
```

Comment:
I should have seen this in #214, but isn't the same logic required on line 250 below?